ThunderCloud/CollectionItemViewCell.swift

```diff
@@ -165,21 +165,24 @@ open class CollectionItemViewCell: UICollectionViewCell {
     /// Calculates the size of the collection list item for the given item
     ///
     /// - Parameter item: The item which will be rendered
+    /// - Parameter includingVerticalPadding: Whether to include the cell's padding in the calculation or not
     /// - Returns: The size the items content will occupy
-    public class func size(for item: CollectionCellDisplayable) -> CGSize {
+    public class func size(
+        for item: CollectionCellDisplayable,
+        includingVerticalPadding: Bool = true
```

Comment:
Should we include the vertical padding now if it's not on the cell but instead on the section inset of the collectionView?

Reply:
So yeah we have two options here:

1. Remove the padding from this entirely and add the padding to both the section inset and to the height calculation in the `Row` implementation (CollectionListItem)
2. Add `includeVerticalPadding` and only set it to `false` for the collection layout delegate method that needs the ACTUAL height of the cells.

2. Is what I went with, but on reflection, perhaps `1.` is better?

Reply:
So I guess this is what I was thinking in my head.
- The `sectionInset` makes sense but is a new thing. 
- The `cellPadding` is part of the cell itself, but this should really be added in the xib. Looking at ThunderCloud it's not adding any spacing in the xib.

I think this solution is fine, but we should prob rename `cellPadding` to `sectionInset`

![IMG_1153](https://user-images.githubusercontent.com/57952587/94809675-da9ec180-03ea-11eb-9952-ac8bd7fa3663.png)

Reply:
That makes total sense, I'll update `cellPadding` to be `sectionInset`!

Reply:
@simonmitchell Added a photo for clarity, took me ages with this new `.heic` file? :P